### ISSUE TITLE
Difference between brand/logo usage/coloring

### ISSUE THREAD

**Issue by peterblazejewicz** [marionettejs/marionettejs.com#191]
There is a clear visual difference between M logos + with text use here and on library repository:

https://github.com/marionettejs/backbone.marionette/blob/master/readme.md

![20150203201012](https://cloud.githubusercontent.com/assets/14539/6027466/b359eaac-abe2-11e4-9995-8ddb0c6f265e.jpg)

http://marionettejs.com/

![20150203200938](https://cloud.githubusercontent.com/assets/14539/6027471/b8ab62a6-abe2-11e4-8825-fbb9eafb7948.jpg)

**Comment by samccone** [comment]
yeah I think this is just a general treatment difference, but good call I think it should be consistent

**Comment by thejameskyle** [comment]
I prefer to use the later.

**Comment by samccone** [comment]
yep +1 later
